### PR TITLE
Migration to add receives_marketing column to the Potential Teachers table

### PR DESCRIPTION
**Pull request by hannahbergam**
Steps I followed:

1. bin/rails generate migration AddReceivesMarketingToPotentialTeacher receives_marketing:boolean
2. bundle exec rake db:migrate
3. Looked over and then committed the three files this updated!

Excellent practice... following Turner's comment I did:
1. bundle exec rake db:rollback STEP=1
2. Updated the migration to include the part about default and null
3. repeated steps 2 and 3 above

## Links

- spec: []()
- jira ticket: https://codedotorg.atlassian.net/browse/ACQ-1155

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: dashboard/db/schema.rb [hunk 1]
```diff
@@ -1485,6 +1485,7 @@
     t.integer "source_course_offering_id"
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
+    t.boolean "receives_marketing"
```
Comment (TurnerRiley) [inline dashboard/db/schema.rb]: Do we want to have a default value and a null value set for this too? (like [here](https://github.com/code-dot-org/code-dot-org/pull/54758/files#diff-bf90ed5047a62239db22703a2b939c6c3ad3c9aecd5859601afd4aa9cfddbb60R1426))

Comment (hannahbergam) [reply dashboard/db/schema.rb]: Oooo yes yes good call. Rolling back, updating and re-running now!

**Review by TurnerRiley — APPROVED** [review]
Looks great!!

**Review by juanmanzojr — APPROVED** [review]
LGTM!

**Pull request merged**